### ISSUE TITLE
Discord bot should not announce activity when it is not public

### ISSUE THREAD

**Issue by JSustronk** [Inter-Actief/amelie#525]
The bot in the Inter-_Actief_ Discord announces all activities when they are saved. However, when creating an activity, it is possible to mark an activity as 'not public'. The discord bot does still announce these activities.

The bot should only announce activities in the following 2 cases:
1. The activity is created, saved and marked as public
2. A previously activity that was not public, is now saved as a public activity

**Comment by WKobes** [comment]
I disagree. The public checkbox is only used to hide an activity from users that are not logged in on the website. While in theory the discord can be entered by people who are not member of IA, I would still consider this not a public broadcast.

You probably misunderstood the checkbox for being able to create activities as a draft, but we do not support that on the website.

**Comment by JSustronk** [comment]
Yes, you are totally right, I misunderstood the meaning of this button. My mistake.

In that case, it would be great to have the possibility to create draft activities (only visible to the organising committee and board) in which you can set everything up (description, signing-up process, sign-up options) before it is released to the public and everybody can see the activity and register. (And obviously creating a draft activity does not trigger the discord bot).

**Comment by WKobes** [comment]
Changed the issue to feature request. Due to the nature of the current activity models we use I mark this as (very) hard-to-fix, maybe even so hard we have to turn this into won'tfix at some point.
